lib/ansible/modules/system/puppet.py

```diff
@@ -32,6 +32,13 @@
   manifest:
     description:
       - Path to the manifest file to run puppet apply on.
+  noop:
+    description:
+      - Override puppet.conf noop mode.
```

Comment:
Will it be obvious to Puppet users when to set this?

Reply:
Yes, it's a standard puppet feature.  If a user has a need to alter noop (no-operation) they'll know what it is.